attributes/default.rb

```diff
@@ -439,4 +439,7 @@
 # To not fill the SSDs with Logs files that nobody access frequently we set the StoragePolicy for the LOGS dir to be default HOT
 default['hopsworks']['hdfs']['storage_policy']['log']         = "HOT"
 
-default['hopsworks']['enable_metadata_designer']              = "false"
\ No newline at end of file
+default['hopsworks']['enable_metadata_designer']              = "false"
+
+#check nodemanager status
+default["hopsworks"]['check_nodemanager_status']              = "false"
```

Comment:
Needs to be added in metadata.rb as well so that it can be overwritten in the cluster definition